movies-search/src/MovieList/StatusBarContainer.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import MovieList from "./MovieList";
+import StatusBar from "./StatusBar";
+import PropTypes from 'prop-types';
+import { fetchAllMovies, SORT_MOVIES, GET_ALL } from '../actions.jsx'
+import { connect } from 'react-redux';
+import styles from './movieList.css'
+
+class StatusBarContainer extends React.Component {
+    state = { isToggleOn: true, sortOption: filters[0] };
+
+    handleSortChange = this.handleSortChange.bind(this);
+
+    componentDidMount() {
+        const { store } = this.context;
```

Comment:
using component context is a tricky thing. you should not rely on that, especially when old context API is officially not supported anymore